### PR TITLE
Copy changes, 3 subtopics of 'Removing content'

### PR DESCRIPTION
**Pull request by shad0wMe0w**
- Copy changes to 'Archive and record content', and 'Create a removal plan'
- Add new partial files: all previous copy was bundled into single markdown files.

Changes made as per UR feedback and Plain English check.

### CODE REVIEW

**Comment by shad0wMe0w** [comment]
@marksantoso I added another topic's changes to the same PR after you reviewed.

**Comment by joolswood** [comment]
I'm still reviewing, but Matt has confirmed a couple of other changes. Are you ok to add them?

- remove `/identify-stakeholders/`
- refactor `/content-strategy/removing-content/` content to remove reference to Step 2 (Identify your stakeholders)

File: content/content-strategy/removing-content/approval-to-remove/wording.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: text/textblock
+section: Watch your wording
+---
+Some agencies and stakeholders avoid using certain words because they come with the idea of risk.
+If so, try to adapt your language. If ‘delete’ and ‘remove’ do not work, use softer wording like ‘retire’, ‘archive’ and ‘decommission’ when explaining your process.
```
Comment (joolswood) [inline content/content-strategy/removing-content/approval-to-remove/wording.md]: Needs another line break if this is a new para.

File: content/content-strategy/removing-content/archive-record-content/archiving-business.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+layout: text/textblock
+section: Archiving for business records
+---
+Your agency may need to keep a record of the various versions of content. If so, you should use a recognised information management system.
+
+A content management system (CMS) or electronic document record management system (EDRMS) are the best ways of capturing your content.
+
+If your content hasn’t been captured in a recognised information management system then you'll need to make a copy some other way.
+
+## [2]If you're only making a minor change
```
Comment (joolswood) [inline content/content-strategy/removing-content/archive-record-content/archiving-business.md]: Should this be a `H3`?

File: content/content-strategy/removing-content/removal-plan/standards.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: text/textblock
+section: Set up standards
+---
+As you start this removal plan, it’s good practise to think about longer term strategy.
+
+You can negotiate standard removal periods for common types of content, such as media releases or industry notices. For example, you might only retain content for current year and previous 2 years, rather than build up a large backlog online.
+Publish your timeframes to remove content internally. This will help you to initiate regular content reviews and to negotiate removal periods.
```
Comment (joolswood) [inline content/content-strategy/removing-content/removal-plan/standards.md]: Needs another line break if a new para.

File: content/content-strategy/removing-content/archive-record-content/archiving-business.md (hunk 2 above)
Comment (shad0wMe0w) [reply content/content-strategy/removing-content/archive-record-content/archiving-business.md]: @joolswood yes this is a `H3`, as per the copy provided.

Comment (joolswood) [reply content/content-strategy/removing-content/archive-record-content/archiving-business.md]: `##` is a `H2`

**Comment by shad0wMe0w** [comment]
@joolswood line breaks fixed. H3 is correct.

**Comment by joolswood** [comment]
@shad0wMe0w if it's a `H3` it needs to be changed to `### [2]`.

`## [2]` will give you H3 display style but still render semantically as a `H2`.

**Comment by joolswood** [comment]
Hey @marksantoso how come you merged? My review wasn't finished?

**Pull request merged**